examples/alpha/dam_basic_deployment/variables.tf

```diff
@@ -20,6 +20,12 @@ variable "gw_count" {
   description = "Number of agent gateways"
 }
 
+variable "agents_count" {
+  type        = number
+  default     = 1
+  description = "Number of databases with agent"
```

Comment:
This description makes me think that there are databases without agent (which there is actually, the Sonar ones, but they are not present in this context, unless your context is all of DSF)